Spanner/src/Backup.php

```diff
@@ -151,12 +151,20 @@ public function __construct(
      *        with microseconds granularity that must be at least 6 hours and
      *        at most 366 days. Once the expireTime has passed, the backup is
      *        eligible to be automatically deleted by Cloud Spanner.
-     * @param array $options [optional] Configuration Options.
+     * @param array $options [optional] {
+     *         Configuration Options.
      *
+     *         @type \DateTimeInterface $versionTime The version time for the externally
+     *              consistent copy of the database. If not present, it will be the same
+     *              as the create time of the backup.
+     *     }
      * @return LongRunningOperation<Backup>
      */
     public function create($database, \DateTimeInterface $expireTime, array $options = [])
     {
+        if (isset($options['versionTime'])) {
+            $options['versionTime'] = $this->pluck('versionTime', $options)->format('Y-m-d\TH:i:s.u\Z');
```

Comment:
This `pluck` is superfluous. Pluck is good when you want to check something exists and remove it from the array. You already know it exists because of the `if`, and it is intended to remain in the array, just in a modified form. We can just do `$options['versionTime']->format()`.

Additionally, you need to check that `$options['versionTime']` is a `DateTimeInterface`, otherwise there's an invalid method error. Not sure if it should be in the `if` on line 165, you may want to raise an exception rather than silently failing. If you raise an exception, be sure to document it with a `@throws` annotation.